src/tracker-component.js

```diff
@@ -251,6 +276,19 @@ export const Tracker = (config? : Config = defaultTrackerConfig) => {
         },
         setProperty: (data : PropertyData) => {
             config.properties = { ...config.properties, ...data };
+        },
+        getIdentity: (data: IdentityData) => {
+            // make url different for different envs later
+            const url = 'https://localhost.paypal.com:8443/muse/api/partner-token'
```

Comment:
this is getting merged into production, so this will not work. One thing you can do is take in an optional url in the `getIdentity` parameter.

Reply:
what should the default url be if they don't pass in a url then?